### ISSUE TITLE
Widget with heights info for indicated nearest airspace, as opposed to current location info

### ISSUE THREAD

**Issue by bluearcus** [fhorinek/SkyDrop#423]
Currently, we have two airspace altitude / height info widgets.

Both are described as relating to the airspace at the current position.

Can we have a widget providing info on the airspace on the other side of the boundary pointed to by the nearest airspace boundary direction arrow / distance widgets?

I want to know about lowering ceiling nearby, or minimum altitude nearby. The current widgets don't seem to support this as described.

**Comment by bubeck** [comment]
Yes, I agree. We should use the direction of the glider to show, what the coming airspace is.

I am willing to implement this, but there are some patches from me waiting for integration and Fero changed a lot in the last (not yet released) version. So I will wait for the next official version and then start on this.

As I am often fly near airspaces (paramotor) I am very interested in getting this done properly. :)

**Comment by bluearcus** [comment]
Direction of the glider... Not so much. Just the nearest airspace direction will do fine. Direction of the glider is confusing, if you are skirting airspace the widget may show something other than what you are avoiding, leading to confusion. Nearest is best. It will show what we are interested in almost all of the time and not skip between one thing and another randomly when thermalling.On 29 Nov 2019 08:07, "Dr. Tilmann Bubeck" <notifications@github.com> wrote:Yes, I agree. We should use the direction of the glider to show, what the coming airspace is.
I am willing to implement this, but there are some patches from me waiting for integration and Fero changed a lot in the last (not yet released) version. So I will wait for the next official version and then start on this.
As I am often fly near airspaces (paramotor) I am very interested in getting this done properly. :)
